libraries/app/application.cpp

```diff
@@ -410,7 +410,7 @@ void application_impl::startup()
    }
 
    if( _options->count("enable-subscribe-to-all") )
-      _app_options.enable_subscribe_to_all = _options->at("enable-subscribe-to-all").as<bool>();
```

Comment:
Sorry, I didn't get it, what's wrong with `_options->at("enable-subscribe-to-all").as<bool>();`?

Reply:
If you don't add the command line option, you get an index out of range error